Lecture-4/Task4/l4task4/Application.java

```diff
@@ -0,0 +1,59 @@
+package l4task4;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Scanner;
+
+public class Application {
+
+    private static List<Runner> runners;
+
+    private static void showListOfRunners(List<Runner> runners) {
+        for (Runner runner : runners) {
+            System.out.println(runner.toString());
```

Comment:
no need to call `toString()` method on a `runner` object - method is called automatically

Reply:
done